Gopkg.toml

```diff
@@ -44,3 +44,45 @@
 [[constraint]]
   name = "github.com/BurntSushi/toml"
   revision = "a368813c5e648fee92e5f6c30e3944ff9d5e8895"
+
+[[constraint]]
+  name = "github.com/go-logfmt/logfmt"
+  version = "0.3.0"
+
+[[constraint]]
+  branch = "v2"
+  name = "gopkg.in/yaml.v2"
+
+[[constraint]]
+  name = "github.com/sirupsen/logrus"
+  version = "1.4.1"
+
+[[constraint]]
+  name = "github.com/montanaflynn/stats"
+  version = "0.2.0"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/olekukonko/tablewriter"
+
+[[constraint]]
+  name = "github.com/stretchr/testify"
+  version = "1.3.0"
+
+[[constraint]]
+  name = "gopkg.in/russross/blackfriday.v2"
+  version = "2.0.1"
+
+[[constraint]]
+  name = "github.com/slok/kubewebhook"
+  version = "0.2.0"
+
+
+[[constraint]]
+  branch = "master"
+  name = "k8s.io/api"
```

Comment:
I was going to say we should pin these to a specific version as these are currently pulled from the master branch, but looks like these were never pinned. 
I am not sure if the tool itself required the latest, something to consider.

Reply:
Agreed. Not sure if this was intentional to make sure we always keep up with latest API changes from k8s. @mcastelino ?